policy/lib/tekton/recorded_att_data_test.rego

```diff
@@ -1,6 +1,6 @@
 # ** Do not edit this file. Regenerate it using `make sync-test-data` **
 
-package lib.tkn.recorded_att_test
+package lib.tekton
```

Comment:
`make sync-test-data` reverts this, needs a corresponding change here: https://github.com/enterprise-contract/ec-policies/blob/cd89ed963a1e664520e7420063adbc3be9c669dd/Makefile#L156

Reply:
I knew that and still forgot to make the change. Thanks.